data/formats-data.js

```diff
@@ -2183,6 +2183,16 @@ exports.BattleFormatsData = {
 		],
 		tier: "New",
 	},
+	smearglepokestar: {
```

Comment:
I... hate the idea of this not being considered an alt-forme of Smeargle, simply because it is one. It's the only one of these that's a "real" Pokemon wearing a costume, and isn't just CGI. However, as long as there's no mechanical reason why this would have to be the same species on sim, I suppose I can change this.